# List of 31 species from EukProt v3 TCS (out of 196) for which I identified phylogenetic evidence of potential eukaryotic/bacterial contamination

EukProt version 3 ‘The Comparative Set’ (TCS) includes 196 species [https://figshare.com/articles/dataset/TCS_tar_gz/21586065]. Based on the contamination screenings done, I recommend excluding these 31 species:

* Potential eukaryotic and bacterial contamination (4): _Choanocystis_sp, Colponema_vietnamica, Leptophrys_vorax, Nonionella_stella_
* Potential eukaryotic contamination (7): _Colponemidia_sp_Colp-10, Colponemidia_sp_Colp-15, Gloeochaete_wittrockiana, Lapot_gusevi, Madagascaria_erythrocladioides, Syssomonas_multiformis, Telonema_sp_
* Potential bacterial contamination (20): _Amoeba_proteus, Amphimedon_queenslandica, Anaeramoeba_flamelloides, Anaeramoeba_ignava, Ancoracysta_twista, Apusomonadida_sp, Barthelona_sp, Capitella_teleta, Codosiga_hollandica, Dysnectes_brevis, Entosiphon_sulcatum, Fabomonas_tropica, Lenisia_limosa, Mantamonas_plastica, Nutomonas_longa, Parvularia_atlantis, Paulinella_chromatophora, Rigifila_ramosa, Stygiella_incarcerata, Symbiodinium_sp_

# Files
* eukProtv3TCStaxa_noEvidenceClearContamination.txt: 165 species from EukProt v3 TCS for which I did not identify clear evidence of contamination
* eukProtv3TCStaxa_yesEvidenceClearContamination.txt: 31 species from EukProt v3 TCS for which I identified clear evidence of contamination
* EukaryoteContaminationInspectedBUSCOphylogenies.zip: In the context of the eukaryote contaminant screening, these PDF files (max 1 per species) include all BUSCO marker phylogenies which had to be inspected (those with >1 sequences for the query species). Species without PDF files did not have any BUSCO marker phylogeny to be inspected.
* BacterialContaminationInspectedBUSCOphylogenies.zip: In the context of the bacterial contaminant screening, these PDF files (max 1 per species) include all BUSCO marker phylogenies which had to be inspected (those with >1 sequences for the query species). Species without PDF files did not have any BUSCO marker phylogeny to be inspected.

# Methodology
I ran BUSCO v5.8.2 twice for each proteome (‘-m proteins’, hmmsearch v3.1 for the alignments), using ‘-l bacteria_odb10’ and ‘-l eukaryota_odb12’ as “lineage dataset” for the analyses intended to identify potential contaminant sequences from Bacteria and from eukaryotes, respectively.

For the identification of proteomes with potential eukaryotic contamination, I took advantage of the fact that BUSCO identifies, by definition, putative single copy orthologs which are only rarely absent in eukaryotic genomes. I thus reasoned that if a given proteome includes sequences from two or more eukaryotes, BUSCO will likely identify at least two or more copies from different BUSCO markers (BUSCO duplicates). The finding of BUSCO duplicates is per se not evidence of contamination, as some species can present paralogs, and proteome data often presents redundant sequences (isoforms, fragmented sequences from assembly errors, etc.). However, if the BUSCO duplicates are much more similar to BUSCO sequences from highly unrelated eukaryotic species than they are to each other, then this pattern is rather suggestive of either horizontal gene transfer cases or potential contamination, and the second possibility is more likely if the pattern is consistently observed among different BUSCO markers. I thus followed a phylogenetic approach to identify potential contaminant proteomes. For each query proteome, I reconstructed the phylogeny of every BUSCO marker in which duplicates, triplicates, etc. where identified in the query proteome. Apart from the sequences from the query proteome, every phylogeny also incorporated the corresponding BUSCO markers identified in a total of 20 target eukaryotic species: _Acanthamoeba castellanii, Arabidopsis thaliana, Aspergillus nidulans, Bodo saltans, Capsaspora owczarzaki, Chlamydomonas reinhardtii, Chondrus crispus, Cyanidioschyzon merolae, Dictyostelium discoideum, Ectocarpus siliculosus, Homo sapiens, Naegleria gruberi, Ostreococcus tauri, Phytophthora infestans, Reticulomyxa filosa, Rhizophagus irregularis, Salpingoeca rosetta, Tetrahymena thermophila, Thecamonas trahens, Vitrella brassicaformis_. This list of target species was set aiming to cover different branches of the eukaryotic phylogeny while prioritizing also the usage of good quality proteome data (complete, non-fragmented, non-redundant, etc.). On the other hand, the number of target species was set as a compromise between phylogenetic resolution, computational burden, and suitability for visual inspection of the large number of resulting phylogenies. To build the phylogenetic trees, I used MAFFT v7.505 for the multiple sequence alignment step, trimAl v1.5.rev0 build[2024-05-27] for alignment trimming (using the ‘-gappyout’ option), and FastTree v2.2 for the phylogenetic inference. The resulting phylogenies were visually inspected to identify potential contaminant sequences.

I followed a similar approach for the identification of proteomes with potential bacterial contamination. Same as for ‘eukaryota_odb12’ in the case of eukaryotes, BUSCO ‘bacteria_odb12’ consist of a set of marker genes which are putative single copy orthologs widely distributed across Bacteria. However, the identification of BUSCO markers from ‘bacteria_odb12’ in eukaryotic proteomes is not necessarily indicative of contamination, since these markers can also have orthologs in eukaryotes. This approach thus also required building phylogenies for each query proteome and for each BUSCO ‘bacteria_odb12’ (unless not required, see below). Each phylogeny included the identified sequences from the query proteome, from the 20 target eukaryotic species listed above, and also from 23 target bacterial proteomes retrieved from the ‘Quest for Orthologs’ Reference Proteomes Release 2024_02: _Aquifex aeolicus, Bacillus subtilis, Bacteroides thetaiotaomicron, Bradyrhizobium diazoefficiens, Chloroflexus aurantiacus, Chlamydia trachomatis, Deinococcus radiodurans, Dictyoglomus turgidum, Escherichia coli, Fusobacterium nucleatum, Geobacter sulfurreducens, Gloeobacter violaceus, Helicobacter pylori, Leptospira interrogans, Mycoplasma genitalium, Mycobacterium tuberculosis, Neisseria meningitidis, Pseudomonas aeruginosa, Rhodopirellula baltica, Streptomyces coelicolor, Synechocystis sp., Thermotoga maritima, Thermodesulfovibrio yellowstonii_. I performed a first step to discard for visualization cases which were unlikely to represent contamination. I used BLAST 2.16.0+ to align all identified markers in our query proteomes to a database including the BUSCO markers identified in the 23 target bacterial proteomes and in the 20 target eukaryotic proteomes described above [-max_target_seqs 150 -evalue 1e-3]. I only considered for phylogenetic reconstruction and visualization those marker sequences for which the best scoring alignment was a bacterial sequence. The non-discarded sequences from this filter were submitted to a phylogenetic reconstruction process using the same software as described in the paragraph above.

# Example of BUSCO phylogenies suggestive of potential bacterial contamination
## _Amphimedon queenslandica_ (Metazoa, Opisthokonta)

![Screenshot from 2025-07-04 17-10-11](https://github.com/user-attachments/assets/f43f0a38-4319-407a-9280-986eca528414)

Case 1: This BUSCO phylogeny includes a sequence from _A. queenslandica_ surrounded by sequences from bacterial origin. BUSCO identified an ortholog in _A. queenslandica_ but not in the other eukaryotes, strongly suggesting a potential bacterial origin/contamination for this sequence.

![Screenshot from 2025-07-04 17-12-04](https://github.com/user-attachments/assets/b63c8124-e368-477b-8fed-2252f0784a91)

Case 2: The sequence from _A. queenslandica_ branches in a very distant clade to the other eukaryotic sequences (non-black colors). The other eukaryotic species represented are plastid-bearing lineages, which are phylogenetically unrelated to _A. queenslandica_. The decision of considering _A. queenslandica_ as a species including potential contamination was taken based on the finding on multiple examples of Case 1 and Case 2.

![Screenshot from 2025-07-04 17-37-31](https://github.com/user-attachments/assets/cc36d8fd-232b-4c80-9b84-acca933f4d33)

Case 3: Despite the sequence from _A. queenslandica_ branches as sister-group to a clade of three bacterial sequences, the low internal branch supports do not allow to confidently infer a bacterial origin/contamination for this sequence. In fact, lack of phylogenetic resolution is the case for the most of BUSCO bacterial markers. Some of them are short-length sequences, which limits the phylogenetic resolution. Others are also highly conserved, which also limits the phylogenetic resolution. We thus were conservative, and only considered as potential evidence for contamination clearer cases such as Case 1 or Case 2.

# Example of BUSCO phylogenies suggestive of potential eukaryotic contamination
## _Syssomonas multiformis_ (Opisthokonta)

![Screenshot from 2025-07-04 17-39-17](https://github.com/user-attachments/assets/0e0a8e5f-8aa1-4621-95e7-cd244f8c0f40)

Case 1: Two sequences from _S. multiformis_ were identified for this BUSCO marker. One of them branches, as expected, with other opisthokonta sequences (from _Rhizophagus irregularis_, a fungal species, and from _Capsaspora owczarzaki_, a unicellular holozoan). In contrast, the other sequence branches with two sequences from Stramenopiles (from _Ectocarpus siliculosus_, a brown algae, and from _Phytophtora infestans_, an oomycete). Stramenopiles are very phylogenetically unrelated to Opisthokonta. Of course, observing one of such a case is per se not conclusive of contamination: deep phylogenetic reconstructions, and particularly single-gene based phylogenies, are sensitive to errors. However, it is by the observance of multiple instances of this same pattern (see the phylogenies below) that potential contamination in _S. multiformis_ (Opisthokonta) from Stramenopiles was inferred. This finding is not surprising considering the fact that this species was fed with _Spumella sp._ (a Stramenopiles species) while being maintained in cultures (REF: https://doi.org/10.1016/j.cub.2017.06.006).

![Screenshot from 2025-07-04 17-43-26](https://github.com/user-attachments/assets/d177e159-85cc-4315-ab55-f3f800cf86c1)

![Screenshot from 2025-07-04 17-46-04](https://github.com/user-attachments/assets/49b998ad-eebb-4df9-be68-b16bc32f1023)

NOTE: Do not pay attention to sequence names, as this were reformatted for standarization purposes in order to facilitate the manual inspection of the phylogenies.


